samples/snippets/csharp/VS_Snippets_CLR/conceptual.disposable/cs/base1.cs

```diff
@@ -6,84 +6,84 @@
 
 public class DisposableStreamResource : IDisposable
 {
-   // Define constants.
-   protected const uint GENERIC_READ = 0x80000000;
-   protected const uint FILE_SHARE_READ = 0x00000001;
-   protected const uint OPEN_EXISTING = 3;
-   protected const uint FILE_ATTRIBUTE_NORMAL = 0x80;
-   protected IntPtr INVALID_HANDLE_VALUE = new IntPtr(-1);
-   private const int INVALID_FILE_SIZE = unchecked((int) 0xFFFFFFFF);
+    // Define constants.
+    protected const uint GENERIC_READ = 0x80000000;
+    protected const uint FILE_SHARE_READ = 0x00000001;
+    protected const uint OPEN_EXISTING = 3;
+    protected const uint FILE_ATTRIBUTE_NORMAL = 0x80;
+    protected IntPtr INVALID_HANDLE_VALUE = new IntPtr(-1);
+    const int INVALID_FILE_SIZE = unchecked((int)0xFFFFFFFF);
```

Comment:
```suggestion
    private const int INVALID_FILE_SIZE = unchecked((int)0xFFFFFFFF);
```

Reply:
Unless samples are explicitly using a "fewest words" style, it seems like they should match the style for the BCL, where access modifiers are always specified.